hbase-protocol-shaded/src/main/protobuf/server/ClusterStatus.proto

```diff
@@ -133,8 +133,8 @@ message RegionLoad {
   /** the most recent sequence Id from cache flush */
   optional uint64 complete_sequence_id = 15;
 
-  /** The current data locality for region in the regionserver */
-  optional float data_locality = 16;
+  /** The current blocks local weight for region in the regionserver */
+  optional uint64 blocks_local_weight = 16;
```

Comment:
Same applies to ClusterStatus. Can we just add `float data_locality_for_ssd` at the end (24th element) and keep `data_locality` and `data_locality_for_ssd` as is?

Reply:
"Can we just add float data_locality_for_ssd at the end" 
You mean blocks_local_weight?

I had thought about it, just lead to some redundancy, but you are right that the compatibility should not be breaked in minor, will fix later.
Thanks.